.circleci/deploy-testnet.sh

```diff
@@ -17,14 +17,16 @@ curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh |
 export NVM_DIR="/opt/circleci/.nvm" && . $NVM_DIR/nvm.sh && nvm install v10.14.1 && nvm use v10.14.1
 export COMMIT_HASH=$(./docker/hash.sh)
 
-curl -O https://s3.eu-central-1.amazonaws.com/boyar-ci/boyar/config.json
+cd .circleci && npm install @orbs-network/orbs-nebula && cd ..
+
+curl -O https://boyar-testnet-bootstrap.s3-us-west-2.amazonaws.com/boyar/config.json
```

Comment:
This should be configurable in all shell scripts and taken from the environment.

Reply:
Agreed. Where would you put this URL ?

Reply:
You can pick an environment variable and inject it into circle ci configuration.